clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -52,6 +52,20 @@ abstract class OpenstackClientProvider {
   }
 
   /**
+   * Create a Spinnaker Server Group (Openstack Heat Stack).
+   * @param stackName
+   * @param heatTemplate
+   * @param parameters
+   * @param disableRollback
+   * @param timeoutMins
+   * @return
+   */
+  void deploy(String stackName, String heatTemplate, Map<String, String> parameters, boolean disableRollback, Long timeoutMins) {
```

Comment:
@derekolk what do you mean? the deploy operation is tested in the DeployOpenstackAtomicOperationSpec